.buildkite/pipelines/periodic-packaging.yml

```diff
@@ -29,7 +29,7 @@ steps:
           provider: gcp
           image: family/elasticsearch-{{matrix.image}}
           diskSizeGb: 350
```

Comment:
I'm also wondering now why we provision such a large disk.

Reply:
Maybe for the extra I/O since these don't run in RAM?

Reply:
We use the same size disk for our platform support jobs so I guess so.